src/syncerd/monero_syncer.rs

```diff
@@ -63,10 +63,23 @@ pub struct Transaction {
 }
 
 impl MoneroRpc {
-    fn new(node_rpc_url: String) -> Self {
+    fn new(node_rpc_url: String, proxy_url: Option<String>) -> Self {
+        let mut client_builder = monero_rpc::RpcClientBuilder::new();
+        if let Some(proxy_url) = proxy_url {
```

Comment:
This is the only line in this patch that takes care of a potential proxy, just want to make sure that the rest is handled in #824 and we don't miss setting the proxy elsewhere.